make/go/dep_protoc_gen_connect_go.mk

```diff
@@ -0,0 +1,20 @@
+# Must be set
```

Comment:
I assume this can live in `make/go` without being removed/overwritten/etc. by makego .. but I'm not 100% sure.

Reply:
Maybe I'll leave it here for now and then create a task / issue in Linear to track it?

Reply:
As with all other files in `make/go`, this should not be managed in repositories other than makego, as denoted by the file headers. This should be created in makego first, then copied over.